bigtop-packages/src/common/apex/do-component-build

```diff
@@ -28,6 +28,9 @@ command -v mvn > /dev/null 2>&1 || {
 # Build apex
 mvn clean package -DskipTests "$@"
 
+#Find hadoop.version from pom.xml
+APEX_HADOOP_VERSION=$(grep -oPm1 "(?<=<hadoop.version>)[^<]+" pom.xml)
```

Comment:
I wanted to use the same. But that would mean excluding hadoop 2.7.3 dependencies and not 2.6.0 dependency.. We want to exclude the one that is referenced by apex. Hence done this way.